src/io/ReplicaGenerator.cpp

```diff
@@ -487,7 +487,10 @@ ReplicaGenerator::readPhaseSpace(ParticleContainer* particleContainer, Domain* d
 							Component* comp = &ptrComponents->at(ptrChangeVec->at(cid));
 							mol.setComponent(comp);
 							mol.setid(++nActID);
-							particleContainer->addParticle(mol);
+							// only add particle if it is inside of the own domain!
+							if(particleContainer->isInBoundingBox(mol.r_arr().data())) {
+								particleContainer->addParticle(mol, true, false);
+							}
 							numAddedParticlesLocal++;
```

Comment:
is it sensible to increase this when the if fails?

Reply:
already checked beforehand, so removed this if :)